### ISSUE TITLE
New cube-visualize

### ISSUE THREAD

**Issue by Co-eus** [TillSchlemmermeier/l3d-controller-software#2]

**Comment by Co-eus** [comment]
it would be nice to have a visualizing tool again, which uses all the routines of our software as far as possible. this we can use to test new animations, debug our code etc. - so that we can do it not only with the real cube!

**Comment by TillSchlemmermeier** [comment]
YES !... i have started a few Months ago to build a simulator with the three.js 3D-Engine for browsers... but the fact that it has to run in a Server Environment is not perfect for our purpose... we should take a look at a python realtime 3d engine e.g. Panda3D.. i will look into this


https://www.panda3d.org/

We should also look into pythons multiprocessing to use pipes to exchange data between applications (Main-Controller with GUI, Send-Module, Visualizer, Sound-To-Light...) to get rid of runtime dependencies. This might also speed up things in general.
